packages/core/src/components/Modal/ModalTopActions/ModalTopActions.tsx

```diff
@@ -17,6 +17,7 @@ const ModalTopActions = ({ renderAction, theme, closeButtonAriaLabel, onClose }:
     <div className={styles.actions} data-no-autofocus={true}>
       {typeof renderAction === "function" ? renderAction(buttonColor) : renderAction}
       <IconButton
+        data-testid={"close-modal-button"}
```

Comment:
add in `packages/core/src/tests/constants.ts`:
```
...
MODAL_NEXT_OVERLAY = "modal-overlay",
MODAL_NEXT_HEADER = "modal-header",
--> MODAL_NEXT_CLOSE_BUTTON = "modal-close-button",
...
```

and then:
```suggestion
        data-testid={ComponentDefaultTestId.MODAL_NEXT_CLOSE_BUTTON}
```

Reply:
Done